src/main/kotlin/nl/info/zac/app/admin/model/RestBrpDoelbindingen.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * SPDX-FileCopyrightText: 2025 INFO.nl
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package nl.info.zac.app.admin.model
+
+import net.atos.zac.admin.model.BrpDoelbindingen
+import net.atos.zac.admin.model.ZaakafhandelParameters
+
+class RestBrpDoelbindingen {
```

Comment:
A `data` class with `val`s is generally preferred over a normal Kotlin class. Is that feasible here? If we are in control of generating instances there is no need for setters (which you won't get with data classes).